setup.py

```diff
@@ -63,43 +63,45 @@
     data_files=get_data_files(data_files_spec),
     packages=find_packages(),
     install_requires=[
-        'autopep8>=1.5.0,<1.5.6',
-        'click>=7.1.1,<8',
+        'autopep8>=1.5.0',
+        'click>=8',  # elyra-ai/elyra#2579
         'colorama',
         'deprecation',
         'entrypoints>=0.3',
         'jinja2>=2.11',
-        'jsonschema>=3.2.0',
-        'jupyter_core>=4.0,<5.0',
+        'jsonschema>=3.2.0,<4.0',  # Cap from kfp
+        'jupyter_core>=4.6.0',
         'jupyter_client>=6.1.7',
         'jupyter-packaging>=0.10',
         'jupyter_server>=1.7.0',
         'jupyterlab~=3.3.0',
         'jupyterlab-git~=0.32',
```

Comment:
Since these are essentially capped, could we add comments indicating why we're capping lab to <= 3.3.x and the git extension to <= 0.x?